onnxruntime/contrib_ops/cuda/moe/ft_moe/epilogue_helpers.h

```diff
@@ -83,10 +83,16 @@ namespace ort_fastertransformer {
 
 struct EpilogueOpBiasSilu {};
 
+struct EpilogueOpNoBiasSilu {};
+
 struct EpilogueOpBiasReLU {};
 
+struct EpilogueOpNoBiasReLU {};
+
 struct EpilogueOpBiasFtGelu {};
 
+struct EpilogueOpNoBiasFtGelu {};
```

Comment:
Please use full name for Ft (Fast?)

Reply:
This file originally is from fastertransformer so I follow the naming convention there.